web/src/main/java/model/Account.java

```diff
@@ -0,0 +1,56 @@
+package model;
+
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Data
+@Entity
+@Table
+public class Account {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne
+    private User user;
+
+    private String nameOfAccount;
+
+    private Integer sumOfAccount;
+
+    public Account() {
```

Comment:
Remove the lines of code from here up until the EOF. This is linked to [this comment. ](https://github.com/svelj/finance_app/pull/1/files#r1578180393)